tests/TestCase/Db/Adapter/MysqlAdapterTest.php

```diff
@@ -146,15 +146,6 @@ public function testHasTableUnderstandsSchemaNotation()
         $this->assertFalse($this->adapter->hasTable('unknown_schema.phinxlog'));
     }
 
-    public function testHasTableRespectsDotInTableName()
-    {
-        $sql = "CREATE TABLE `discouraged.naming.convention`
-                (id INT(11) NOT NULL)
-                ENGINE = InnoDB CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci";
-        $this->adapter->execute($sql);
-        $this->assertTrue($this->adapter->hasTable('discouraged.naming.convention'));
```

Comment:
I don't see a reason to continue supporting table names with`.` as they aren't well supported by cakephp/database either.